drivers/rpmsg/Makefile

```diff
@@ -9,4 +9,6 @@ obj-$(CONFIG_RPMSG_QCOM_SMD)	+= qcom_smd.o
 obj-$(CONFIG_RPMSG_VIRTIO)	+= virtio_rpmsg_bus.o
 
 obj-$(CONFIG_RPMSG_RPC)		+= rpmsg-rpc.o
+obj-$(CONFIG_RPMSG_GPIOCHIP_INTERFACE)    += rpmsg_gpio.o
```

Comment:
Please combine this patch with the source code patch. I don't see any reason for them to be separate.